### ISSUE TITLE
Make less ugly

### ISSUE THREAD

**Issue by Rumperuu** [Rumperuu/Pinpointer#4]

**Comment by dufferzafar** [comment]
Yeah, the concept itself is pretty amazing, but the UI could really use some work.

I think some inspiration should be taken from UBlock's Block Element Picker: https://github.com/gorhill/uBlock/wiki/Element-picker

It is GPL licensed, so you can also copy the code if you want.

**Comment by jimbo1qaz** [comment]
I found the green outline and background too prominent, and the background reduces readability.
